src/VisualStudio/Setup/UnifiedSettings/csharpIntellisenseSettings.registration.json

```diff
@@ -0,0 +1,226 @@
+{
+  "properties": {
+    "textEditor.csharp.intellisense.triggerCompletionOnTypingLetters": {
```

Comment:
Can this be made more concise without losing descriptive power?  I suggest replacing triggerCompletionOnTypingLetters with completeOnTyping.  Or is the "letters" part important?

Reply:
Internally we have another option called `triggerCompletionOnTyping` https://github.com/dotnet/roslyn/blob/03a35ec77e70edfc64be862fac0019aa3a47805c/src/Features/LanguageServer/Protocol/Features/Options/CompletionOptionsStorage.cs#L40

So this is called 'triggerCompletionOnTypingLetters' to avoid the misunderstanding.

Reply:
I don't know if `dotnet_trigger_completion_on_typing` could be removed. I just see it is set in unit tests. Maybe it is used by other friend repo? Ping @genlu  for knowledge here.